lib/ClangImporter/ImportDecl.cpp

```diff
@@ -2546,24 +2546,45 @@ namespace {
       return result;
     }
 
+    enum RetainReleaseOperatonKind {notAfunction, doesntReturnVoid, invalidParameters, valid};
```

Comment:
Could you please use `enum class` instead of `enum` here to avoid making `notAfunction` etc. globally available variables.